features/support/specialist_sector_helper.rb

```diff
@@ -1,7 +1,19 @@
 require 'gds_api/test_helpers/content_api'
+require "gds_api/test_helpers/content_store"
+
+Before do
```

Comment:
That's how you write before hooks in Cucumber. https://www.relishapp.com/cucumber/cucumber/docs/writing-support-code/before-hook